tests/wpt/include.ini

```diff
@@ -48,7 +48,7 @@ skip: true
   [css-highlight-api]
     skip: true
   [css-inline]
-    skip: true
```

Comment:
We should probably keep skipping the `initial-letter` and `text-box-trim` subdirectories.

So I would go with

```
  [css-inline]
    skip: true
    [baseline-source]
      skip: false
```

or

```
  [css-inline]
    skip: false
    [initial-letter]
      skip: true
    [text-box-trim]
      skip: true
```

There's also few tests in `animation`, `parsing`, and at the root. I don't think it matters much if they are enabled or not, but if enabled you will need to update the expectations.

Reply:
I will stick with the 1st option